SETUP/devex/dpdev-docker/Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM php:8.1-apache
+
+# Install dependency packages
+RUN apt-get update
+# DP deps: mariadb-client gettext aspell aspell-fr pngcheck
+# Handy dev deps: unzip git
+RUN apt-get install -y mariadb-client gettext aspell aspell-fr pngcheck git unzip
+
+# Install PHP extensions
+ADD --chmod=0755 https://github.com/mlocati/docker-php-extension-installer/releases/latest/download/install-php-extensions /usr/local/bin/
+RUN install-php-extensions intl mbstring memcached gettext mysqli xml zip
+
+# And composer
+COPY install_composer.sh /tmp
+RUN bash /tmp/install_composer.sh && mv composer.phar /usr/local/bin/composer && rm /tmp/install_composer.sh
+
+# Install nodejs/npm courtesy of https://deb.nodesource.com/
+RUN curl -fsSL https://deb.nodesource.com/setup_22.x | bash - && apt-get install -y nodejs
```

Comment:
should we use add here? https://docs.docker.com/reference/dockerfile/#add not sure why it would be better but maybe more idiomatic? idk

Reply:
ADD can copy a file into place, but we'd need to put a RUN after it to run the script (and then to install nodejs). Doing that here is adding another command (but I'm guessing you meant this to be in conjunction with the other suggestion to combine this apt install into the earlier command).